### ISSUE TITLE
Starter code files already populated

### ISSUE THREAD

**Issue by stevehanstudio** [raman-at-pieces/youtube-bookmarker-starter-code#1]
Just started following along the video tutorial.  contentScript.js already has code, and I believe it should start as an empty file since this is the starter code as shown in the video.

**Comment by cosmicqbit** [comment]
Exactly, it's not really a big deal but I was also expecting a blank slate, I mean that is why we had a starter file in the first place.

**Comment by freddymercury** [comment]
same,..i noticed the title of the project is different as well; seems like the starter is the staring point that is preferred. I am going to just locally delete the source code so I can follow the vids. thanks!
